workers/ohsome_quality_analyst/ohsome/client.py

```diff
@@ -97,61 +122,89 @@ async def query_ohsome_api(url: str, data: dict, headers: dict = {}) -> dict:
 
 
 async def get_latest_ohsome_timestamp() -> datetime.datetime:
-    """Get unix timestamp of ohsome from ohsome api."""
-    url = "https://api.ohsome.org/v1/metadata"
-    headers = {"user-agent": USER_AGENT}
+    """Get latest unix timestamp from the ohsome API."""
     async with httpx.AsyncClient() as client:
-        resp = await client.get(url, headers=headers)
-    timestamp_str = str(resp.json()["extractRegion"]["temporalExtent"]["toTimestamp"])
-    timestamp = datetime.datetime.strptime(timestamp_str, "%Y-%m-%dT%H:%MZ")
-    return timestamp
+        resp = await client.get(
+            url=OHSOME_API.rstrip("/") + "/metadata",
+            headers={"user-agent": USER_AGENT},
+        )
+    strtime = resp.json()["extractRegion"]["temporalExtent"]["toTimestamp"]
+    return datetime.datetime.strptime(strtime, "%Y-%m-%dT%H:%MZ")
```

Comment:
Why did you remove the string conversion? `strptime` only takes strings. Do you make sure anywhere else, that the response only contains strings?

Reply:
No, I did not. I just thought that `toTimestmap` is always as string.